src/views/UI/index.tsx

```diff
@@ -389,6 +395,63 @@ export const UI: FC = () => {
           <Tooltip icon={AlertCircle} className="mr-3" text="Tooltip text" />
         </div>
       </div>
+      <div className="mb-4">
+        <Typography className="mb-4" element="h1">
+          Results
+        </Typography>
+        <div className="f f-direction-row f-ai-end mb-3">
```

Comment:
Very nice!
I would suggest to delete the `align-items: flex-end;`, it makes that every result will be the same height on the same line. I think it is possible to achieve the same thing doing something cleaner, but i do not have it on the top of my head right now

```suggestion
        <div className="f f-direction-row mb-3">
```